capstone/src/main/java/com/google/univiz/scorecard/URLProviderImpl.java

```diff
@@ -27,17 +36,43 @@
             .map(SerializedName::value)
             .sorted()
             .collect(toCollection(LinkedHashSet::new));
+    fieldsSugg =
+        Arrays.stream(SuggestionData.class.getDeclaredMethods())
+            .map(field -> field.getAnnotation(SerializedName.class))
+            .filter(Objects::nonNull)
+            .map(SerializedName::value)
+            .sorted()
+            .collect(toCollection(LinkedHashSet::new));
```

Comment:
This is now duplicated code. Consider pulling out a helper method:

```java
  private static Set<String> getCollect(Class<?> serilizableClass) {
    return Arrays.stream(serilizableClass.getDeclaredMethods())
        .map(field -> field.getAnnotation(SerializedName.class))
        .filter(Objects::nonNull)
        .map(SerializedName::value)
        .sorted()
        .collect(toCollection(LinkedHashSet::new));
  }
```

Reply:
+1

Reply:
done!